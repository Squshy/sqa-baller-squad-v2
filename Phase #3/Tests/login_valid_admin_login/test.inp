login
ADMINBOY
AdPass123
logout
exit