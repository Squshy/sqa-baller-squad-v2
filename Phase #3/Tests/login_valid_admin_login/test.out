=========================================================
                        LOGIN
=========================================================
Enter your username: ADMIN BOY

Enter your password: AdPass123
---------------------------------------------------------
                Welcome admin, ADMIN BOY!
=========================================================